cmd/vic-machine/delete/delete.go

```diff
@@ -143,6 +143,16 @@ func (d *Uninstall) Run(cli *cli.Context) error {
 		return err
 	}
 
+	if validator.IsVC() {
+		log.Infoln("Removing VCH vSphere extension")
+		if err = executor.GenerateExtensionName(vchConfig); err != nil {
+			return err
```

Comment:
We should continue uninstallation even if the extension unregistration failed. You can simply warning the error, and continue